src/judoscale/core/adapter_api_client.py

```diff
@@ -9,8 +9,10 @@
 class AdapterApiClient:
     def post_report(self, report):
         try:
-            logger.debug("Posting metrics to Judoscale adapter API")
-            requests.post(config.api_base_url + "/v1/metrics", timeout=5, json=report)
+            url_metrics = config.api_base_url + "/v1/metrics"
+            logger.debug(
+                f"Posting metrics {report} to Judoscale adapter API {url_metrics}")
```

Comment:
Shall I keep this explicit debug info that shows what is actually is posted and where?

Reply:
The output now is what this method returns https://github.com/judoscale/judoscale-python/blob/main/src/judoscale/core/reporter.py#L38
`DEBUG - [Judoscale] Posting metrics {'dyno': 'web.1', 'pid': 28438, 'config': {'log_level': 'DEBUG', 'report_interval_seconds': 2}, 'metrics': [[1658494001, 7.68, 'queue_time', None]]} to Judoscale adapter API https://judoscale-python.requestcatcher.com/v1/metrics`
It is indeed verbose. Where can I fetch this number of how many metrics? The `metrics` key has these data:     datetime -> as timestamp
    value -> the request time
    queue_name -> None (we don't get a queue_name, is that bcz we don't have celery jobs yet in place?)
    measurement -> "queue_time"
For debugging I think it would be helpful to leave also in the output which pid (aka process/worker) is reporting.

Reply:
> Where can I fetch this number of how many metrics?

`report["metrics"]` is a list, so I would just want the size of that list.

> we don't get a queue_name, is that bcz we don't have celery jobs yet in place?

Correct! If there's no queue name, we assume it to be _web queue time_, not _job queue time_.

> For debugging I think it would be helpful to leave also in the output which pid (aka process/worker) is reporting.

I agree. That was one of the logging improvements I had added to https://github.com/judoscale/judoscale-python/issues/7.

Reply:
That's a valid point. The thing is that the thread starts with the application preload, and then once gunicorn starts the worker this thread is stopped. I added some logging info on the thread itself to verify:

Reply:
But we should check if thread is alive inside the posting of the metrics so we don't get these logs of _posting 0 metrics..._